housekeeping/docker/kagome/build_and_push.sh

```diff
@@ -37,7 +42,7 @@ if [ "$BUILD_TYPE" = "Custom" ]; then
   COMMIT_HASH="$(git rev-parse --short HEAD)"
   TAG="$DOCKER_USERNAME/kagome:$COMMIT_HASH"
 else
-  TAG="soramitsu/kagome:$VERSION"
+  TAG="soramitsu/kagome:devops"
```

Comment:
Forgot?

Reply:
Nope, prev one does not work, and only use case is testing CI in branches, which is about devops tasks